docs/cristin-proxy-swagger.yaml

```diff
@@ -2067,6 +2067,99 @@ components:
           $ref: '#/components/schemas/Languages'
         code:
           type: string
+    Approvals:
+      type: array
+      required: [ date, authority, status ]
+      items:
+        type: object
+        properties:
+          type:
+            type: string
+          date:
+            type: string
+            format: date-time
+            description: 'Date of approval'
+          authority:
+            type: string
+            description: 'Approval authority. Only certain combinations of authority and applicationCode are allowed. 
+            RegionalEthicalCommittees: Regional committees for medical and health research ethics (en), 
+            Regionale komitéer for medisinsk og helsefaglig forskningsetikk (REK) (nb), 
+            Regionale komitéer for medisinsk og helsefaglig forskningsetikk (REK) (nn), 
+            applicationCode: EthicalApproval or BioBank; 
+            NorwegianAnimalResearchAuthority: The Norwegian Animal Research Authority (en), Forsøksdyrutvalget (nb), Forsøksdyrutvalet (nn), 
+            applicationCode: AnimalTesting; 
+            NorwegianDataProtectionAuthority: The Norwegian Data Protection Authority (en), Datatilsynet (nb), Datatilsynet (nn), 
+            applicationCode: SensitiveInformation; 
+            NorwegianMedicinesAgency: Norwegian Medicines Agency (en), Statens legemiddelverk (nb), Statens legemiddelverk (nn), 
+            applicationCode: DrugTrial; 
+            NorwegianDirectorateOfHealth: Norwegian Directorate of Health (en), Helsedirektoratet (nb), Helsedirektoratet (nn), 
+            applicationCode: BioTechnology or MedicalEquipment.'
+            enum:
+              - RegionalEthicalCommittees
+              - NorwegianAnimalResearchAuthority
+              - NorwegianDataProtectionAuthority
+              - NorwegianMedicinesAgency
+              - NorwegianDirectorateOfHealth
+          status:
+            type: string
+            description: 'Status of the application sent to the approval authority.'
+            enum:
+              - NotApplied
+              - Applied
+              - Approved
+              - Declined
+              - Rejection
```

Comment:
Two things, one of which is likely irrelevant: Rejection is a noun, the rest are adjectives (rejected), less relevant — how is declined different from rejected?

Reply:
I dont know what the difference is, but it might have to do with an application is not formatted correctly, while the other is formatted correctly but not approved. If you want I can ask someone in Cristin.

By the way, this is not the scope of this PR, but I can inquire if you want